src/api/ums/umsModule.ts

```diff
@@ -139,6 +161,69 @@ export class UMSModule implements IModule {
       ));
   }
 
+  /**
+   * Select users from UMS
+   *
+   * @example
+   * ums.select().where(
+   *   field => field("active").isEqualTo(true)
+   * ).subscribe();
+   *
+   * @returns SelectQuery
```

Comment:
I think `compodoc` gets the return type from the method signature, could you check it?

Reply:
Yeah, you're right, fixed it